app/services/catalog/tenant_settings.rb

```diff
@@ -0,0 +1,24 @@
+module Catalog
+  class TenantSettings
+    attr_reader :response
+
+    def initialize(tenant)
+      @tenant = tenant
+    end
+
+    def process
+      @response = {
```

Comment:
This seems like an area that could be pulled into a concern (and the `schema` method on line 20, but I believe as of now this is the only service class using it.

Reply:
Yes, I could see that being pulled into a base class especially if we start doing this for multiple types. We can split it up later if/when we need to.